.github/workflows/docker_merge.yml

```diff
@@ -50,10 +54,16 @@ jobs:
       - name: Merge and push the per-platform images
         working-directory: ${{ runner.temp }}/digests
         run: |
-          docker buildx imagetools create \
-            -t firedrakeproject/${{ inputs.target }}:${{ inputs.tag }} \
-            -t firedrakeproject/${{ inputs.target }}:latest \
-            $(printf 'firedrakeproject/${{ inputs.target }}@sha256:%s ' *)
+          if [[ "${{ inputs.tag_latest }}" == "true" ]]; then
```

Comment:
Is this line tested? Seems odd to test a boolean value using a string comparison.

Reply:
github actions think that `inputs.tag_latest` is a bool, but bash thinks it is a string.